test/package.json

```diff
@@ -12,5 +12,11 @@
     "playwright": "^1.8.0",
     "supertest": "^6.1.1",
     "ts-jest": "^26.4.4"
+  },
+  "dependencies": {
+    "jest-playwright-preset": "^1.5.1",
+    "playwright-chromium": "^1.10.0",
+    "playwright-firefox": "^1.10.0",
+    "playwright-webkit": "^1.10.0"
```

Comment:
I think it may make sense to move the others to devDependencies too

Reply:
Yes! Good catch